### ISSUE TITLE
MatAutoComplete: Style and Class should affect TextField

### ISSUE THREAD

**Issue by dga711** [SamProf/MatBlazor#243]
After hitting this problem and remembering #240 i think setting the Style property on MatAutoComplete should affect the MatTextField inside it.
Otherwise there is no way to really customize the input field.

**Comment by joelbyrd** [comment]
> After hitting this problem and remembering #240 i think setting the Style property on MatAutoComplete should affect the MatTextField inside it.
> Otherwise there is no way to really customize the input field.
> 
> Same goes for MatTextField itself. I think it is of no use the style the DIVs that wrap the controls

I disagree. If there's only going to be a single component where style and classes can be affected, it *needs* to be the outer most element - that way using CSS selectors (specifically child selectors, you can style the container *and anything inside it* (e.g. `.my-class > input {...}`. Another option is to have multiple style/class attributes, one for each element used in DOM.
